### PR TITLE
Display count of all job runs on show page

### PR DESCRIPTION
**Pull request by rupurt**

### CODE REVIEW

File: gui/__tests__/selectors.test.js [hunk 1]
```diff
@@ -125,6 +126,30 @@ describe('selectors', () => {
...
+    it('returns the number of runs for the job', () => {
+      const state = {
+        jobs: {
+          items: {
+            jobA: {id: 'jobA', runs: ['runA', 'runB', 'runC', 'runD', 'runE', 'runF']}
+          }
+        },
+        jobRuns: {
+          items: {
+            'runA': {id: 'runA'},
```
Comment (j16r) [inline gui/__tests__/selectors.test.js]: Do we need A through F? Wouldn't A + B be sufficient?

Comment (rupurt) [reply gui/__tests__/selectors.test.js]: It shows that the count can be > 5 which is the limit of `latestJobRuns`

Comment (rupurt) [reply gui/__tests__/selectors.test.js]: How do you feel about updating the test name so it's something like:

```javascript
it('returns the run total for the job, not just the total of latest runs', () => {
   // ...
})
```

Comment (j16r) [reply gui/__tests__/selectors.test.js]: Could it be configured to paginate at 1?

Comment (rupurt) [reply gui/__tests__/selectors.test.js]: Currently there's no ability to paginate that relationship

Comment (j16r) [reply gui/__tests__/selectors.test.js]: Right, I'm just suggesting we inject the value `5` and set it to something like `1` for testing. This way we don't have to rewrite the tests if we decide to change it, and the tests can be shorter.

**Pull request merged**